### ISSUE TITLE
UI Error: bad argument #1 to 'unpack'

### ISSUE THREAD

**Issue by brandonrdn** [DakJaniels/LuiExtended#194]
Description:
I'm getting a UI Error when hit with a `Throw Dagger`. 

Error: 
```
bad argument #1 to 'unpack' (table/struct expected, got nil)
stack traceback:
[C]: in function 'unpack'
user:/AddOns/LuiExtended/modules/CombatText/EventViewers/Base.lua:245: in function
'CombatTextEventViewer:PrepareLabel'
<Locals> self = [table:1]{}, label = ud, fontSize = 10, text = "Throw Dagger 177", Settings = [table:2]{} </Locals>
user:/AddOns/LuiExtended/modules/CombatText/EventViewers/CombatScroll.lua:74: in function
'CombatTextCombatScrollEventViewer:View'
<Locals> self = [table:1], combatType = "LUIE_CombatText_COMBAT_TYPE_IN...", powerType = 0, value = 177, abilityName = "Throw Dagger", abilityId = 28502, damageType = 12, sourceName = "Maulborn Recruit^M", isDot = T, hits = 1, Settings = [table:2], controlPoolKey = 51, textFormat = "%t $a", fontSize = 10 </Locals>
user:/AddOns/LuiExtended/modules/CombatText/EventViewers/CombatScroll.lua:57: in function
'CombatTextCombatScrollEventViewer:ViewFromEventBuffer'
<Locals> self = [table:1], combatType = "LUIE_CombatText_COMBAT_TYPE_IN...", powerType = 0, eventKey = "LUIE_CombatText_COMBAT_TYPE_IN...", abilityName = "Throw Dagger", abilityId = 28502, damageType = 12, sourceName = "Maulborn Recruit^M", isDot = T, value = 177, hits = 1 </Locals>
user:/AddOns/LuiExtended/modules/CombatText/EventViewers/CombatScroll.lua:42: in function 'func'
EsoUI/Libraries/Globals/globalapi.lua:222: in function '(anonymous)'
```

Screenshot:
![Screenshot_20210308_165124](https://user-images.githubusercontent.com/25042340/110387248-53a3e400-802f-11eb-8d49-203a2b5ac410.png)

To Reproduce:
Get hit by a `Throw Dagger`. I am currently in Deshaan, near Obsidian Gorge.

Other Installed AddOns:
Action Duration Reminder
CombatMetrics
Dressing Room
Inventory Insight
Journal Quest Log
Khrill Merlin The Enchanter
Map Pins
S'rendarr
Tamriel Trade Centre

**Comment by brandonrdn** [comment]
Looks like this issue was fixed with version 6.2.4. Closing

**Comment by ArtOfShred** [comment]
Thanks for reporting, and for closing out the issue!
